sfdc/Package.md

```diff
@@ -1,501 +1,22 @@
-Connects to Salesforce from Ballerina.
+Connects to Salesforce from Ballerina
 
-## Module Overview
-The Salesforce connector allows users to perform CRUD operations for SObjects, query using SOQL, search using SOSL, 
-and describe SObjects and organizational data through the Salesforce REST API. Also, it supports adding bulk data jobs 
-and batches of types JSON, XML, and CSV via the Salesforce Bulk API. Apart from these functionalities Ballerina 
-Salesforce Connector includes a listener module to capture events. This connector follows OAuth 2.0 authentication for 
-secure access. 
+## Package overview
+The `ballerinax/sfdc` is a [Ballerina](https://ballerina.io/) connector for Google Drive.
```

Comment:
This should be `Salesforce`. And also we need to tell we have both client and listener capabilities.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/bfb4a1f6564fb7ec7fe75e716478e449cb6c9f8c